test/integration/users_login_test.rb

```diff
@@ -0,0 +1,42 @@
+require 'test_helper'
+
+class UsersLoginTest < ActionDispatch::IntegrationTest
+  def setup
+    @user = users(:michael)
+  end
+
+  test "POST login with invalid login credentials" do
```

Comment:
```suggestion
  test "POST login with invalid login credentials should ..." do
```
add what the main assertions are to test name, ditto for other tests in this file